src/interfaces/keystore.ts

```diff
@@ -103,7 +103,10 @@ export type InternalKey = {
   type: 'internal'
   label: string
   dedicatedToOneSA: boolean
-  meta: null
+  meta: {
+    createdAt: number | null
+    [key: string]: any
```

Comment:
For internal keys, there won't be any other attr than `createdAt`, no?

Reply:
Synced up with Simo and we agreed to remove it https://github.com/AmbireTech/ambire-common/pull/977/commits/3c84c61bbcdecd87df6f31c9df32fa501d6ef5e3